setup.py

```diff
@@ -20,6 +20,7 @@
     author='Ryan Tilder',
     author_email="service-dev@mozilla.com",
     url="http://mozilla.org",
+    install_requires=["M2Crypto"],
```

Comment:
Correct, we use --no-deps, https://github.com/oremj/fabdeploytools/blob/master/fabdeploytools/helpers.py#L33